src/internal/m365/collection/site/collection_test.go

```diff
@@ -233,9 +237,15 @@ func (suite *SharePointCollectionSuite) TestCollection_Items() {
 			info, err := shareInfo.Info()
 			require.NoError(t, err, clues.ToCore(err))
 
-			assert.NotNil(t, info)
-			assert.NotNil(t, info.SharePoint)
-			assert.Equal(t, test.itemName, info.SharePoint.ItemName)
+			require.NotNil(t, info)
+			require.NotNil(t, info.SharePoint)
+
+			if test.scope.Category().String() == selectors.SharePointList.String() {
+				require.NotNil(t, info.SharePoint.List)
+				assert.Equal(t, test.itemName, info.SharePoint.List.Name)
```

Comment:
Didn't know we were not using the original `ItemName`.

Reply:
@meain  for lists only populating 'name' of [ListInfo](https://github.com/alcionai/corso/blob/43ca43910fdb241a8ae5d917a6500bcea274cab4/src/pkg/services/m365/api/lists.go#L623) and not the itemName in SharepointInfo

Reply:
yeah. Let me make the change to populate `itemName` name instead.

Reply:
No need to change it if this has already been accepted to be how we do it. I was just thinking out loud.